core/controllers/reader.py

```diff
@@ -79,6 +79,48 @@ def _does_exploration_exist(exploration_id, version, collection_id):
 class ExplorationEmbedPage(base.BaseHandler):
     """Page describing a single embedded exploration."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'v': {
+                'schema': {
+                    'type': 'int',
+                    'validators': [{
+                        'id': 'is_at_least',
+                        'min_value': 1
+                    }]
+                },
+                'default_value': None
+            },
+            'collection_id': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_regex_matched',
+                        'regex_pattern': constants.ENTITY_ID_REGEX
+                    }]
+                },
+                'default_value': None
+            },
+            'iframed': {
+                'schema': {
+                    'type': 'bool'
+                },
+                'default_value': None
```

Comment:
Why is the `default_value` `None`?

Reply:
Because I saw this particular code: https://github.com/oppia/oppia/blob/f4bc55191cc1d1421e5f5f159346fc78c5360f9f/core/controllers/reader.py#L100-L102
and I thought it is possible that no value for `iframed` can be passed. I didn't know if I could give the `default_value` as `False` which is why I gave it `None`

Reply:
Then why don't you use `False`?

Reply:
Done. I was not sure.